### PR TITLE
Adds auth to documented endpoints

### PR DESCRIPTION
**Pull request by ryanlabouve**

### CODE REVIEW

File: lib/api-utils.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { NextApiRequest } from "next";
+
+export const isAuthorized = ({ req }: { req: NextApiRequest }): boolean => {
+  const serverAuthToken = req.headers["x-server-auth"];
+  return !serverAuthToken || serverAuthToken !== process.env.SERVER_AUTH_TOKEN;
+};
```
Comment (johnmosesman) [inline lib/api-utils.ts:6]: Should the name be `isNotAuthorized` based on how the if is written?

File: pages/api/v1/employees.ts [hunk 2]
```diff
@@ -39,6 +40,10 @@ export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse
 ) {
+  if (!isAuthorized({ req })) {
```
Comment (johnmosesman) [inline pages/api/v1/employees.ts:43]: The swagger doc for these endpoints needs the auth token info added to it (like in /deparments)

File: pages/api/v1/departments.ts [hunk 3]
```diff
@@ -55,9 +56,7 @@ export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse
 ) {
-  const serverAuthToken = req.headers["x-server-auth"];
-
-  if (!serverAuthToken || serverAuthToken !== process.env.SERVER_AUTH_TOKEN) {
+  if (!isAuthorized({ req })) {
```
Comment (johnmosesman) [inline pages/api/v1/departments.ts:59]: based on above comment, don't need `!` ?

File: package.json [hunk 4]
```diff
@@ -7,7 +7,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "typecheck": "tsc -b"
+    "typecheck": "tsc -b",
+    "reseed": "npx prisma migrate reset --force && npx ts-node -O '{\"module\": \"commonjs\"}' prisma/seed.ts"
```
Comment (ryanlabouve) [inline package.json:11]: Unrelated, but small improvement 🌲

**Review by johnmosesman — APPROVED** [review]

**Pull request merged**